pyvips/voperation.py

```diff
@@ -280,8 +280,8 @@ def add_reference(x):
             details = intro.details[name]
 
             if (details['flags'] & _DEPRECATED) != 0:
-                logger.info('{0} argument {1} is deprecated',
-                            operation_name, name)
+                logger.info('{0} argument {1} is deprecated'
+                            .format(operation_name, name))
```

Comment:
Perhaps it would be better to do this?:
```suggestion
                logger.info('%s argument %s is deprecated'
                            operation_name, name)
```

Reply:
You're right, that's a bit quicker since you only pay the string interpolation cost if info is turned on.